ui-particles-angular/projects/ui-particles-angular/src/lib/gio-confirm-dialog/gio-confirm-dialog.component.html

```diff
@@ -21,6 +21,14 @@ <h2 mat-dialog-title class="confirm-dialog__title">{{ title }}</h2>
   <p [innerHTML]="content"></p>
 </mat-dialog-content>
 
+<mat-dialog-content *ngIf="contentComponentOutlet && !contentComponentInjector" class="confirm-dialog__content">
```

Comment:
🤔 We always have contentComponentInjector right ?

Reply:
Now yes